packages/manager/modules/mfa-enrollment/src/index.js

```diff
@@ -0,0 +1,31 @@
+import angular from 'angular';
+import 'angular-translate';
+
+import component from './mfa-enrollment.component';
+import routing from './mfa-enrollment.routes';
+import './mfa-enrollment.less';
+
+const moduleName = 'ovhManagerMfaEnrollment';
+let alreadyShowMFA = false;
+
+angular
+  .module(moduleName, [
+    'pascalprecht.translate',
+  ])
+  .component('mfaEnrollment', component)
+  .config(routing)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(($q, $state, OvhApiAuth) => OvhApiAuth.v6()
```

Comment:
Maybe could add explicit ngInject comment:

```suggestion
  .run(/* @ngInject */($q, $state, OvhApiAuth) => OvhApiAuth.v6()
```